packages/ui/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [2.0.0-rc.1]
+
+### Added
+
+- `ui/VirtualList` support `data-webos-voice-focused` and `data-webos-voice-group-label`
```

Comment:
Please use `ui/VirtualList.VirtualList` and `ui/VirtualList.VirtualGridList` instead of `ui/VirtualList`.

Reply:
OK~ I will fix it~